amazon-cloudwatch/fluentd-config-map.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+data:
+  # Configuration is in Json format. No matter what configure change you make,
+  # please keep the Json blob valid.
+  # FIXME FIXME FIXME replace name and region
+  cluster-info.json: |
+    {
+      "cluster": {
+        "name": ""
+      },
+      "logs": {
+        "region": ""
```

Comment:
I believe this could be injected by `eksctl` as we probably read this piece of metadata either from the CLI flags, the `ClusterConfig` object, or AWS?

Reply:
yes, we need to add this to the code, actually